### ISSUE TITLE
Misconfigured -- Unsupported OS when doing setName(Browser.Safari)

### ISSUE THREAD

**Issue by nadvolod** [saucelabs/simple_sauce#145]
```
        SauceOptions options = new SauceOptions();
        options.setName(name.getMethodName());
        options.setBrowserName(com.saucelabs.saucebindings.Browser.SAFARI);
```
That code used in a solution throws the following error:
```
Misconfigured -- Unsupported OS/browser/version/device combo: OS: 'Windows 10', Browser: 'safari', Version: 'latest', Device: 'unspecified' (WARNING: The server did not provide any stacktrace information)
```

**Comment by picimako** [comment]
@nadvolod 
What would be the expected outcome here?
I guess there are other unsupported combinations of configuration properties. Should they also be validated?

Edit:
I think this issue may be covered by https://github.com/saucelabs/sauce_bindings/issues/39

**Comment by titusfortner** [comment]
This issue is mostly solved with #141 but #39 also applies. Both of those apply to all languages.
I'm closing this one and we can track progress in those other two.
